python/ray/_private/utils.py

```diff
@@ -1396,7 +1396,12 @@ def get_runtime_env_info(
 
     proto_runtime_env_info = ProtoRuntimeEnvInfo()
```

Comment:
In this PR, the `RuntimeEnv` protobuf is removed, but `RuntimeEnvInfo` remains as a protobuf.  Do we plan to remove the `RuntimeEnvInfo` protobuf, or is it not necessary?  I guess it needs to be a protobuf because it's sent around between Ray Core processes, is that right?

Reply:
Yep, I think we should keep this protobuf. It is not related to the pluggability.